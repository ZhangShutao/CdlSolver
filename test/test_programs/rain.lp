% Program Reading
% WA 1: <{cloudy},{cloudy}>,
%       <{rainy, prepare_rain_gear},{rainy, prepare_rain_gear}>
% WA 2: <{cloudy, prepare_rain_gear},{cloudy, prepare_rain_gear}>,
%       <{rainy, prepare_rain_gear},{rainy, prepare_rain_gear}>

:- &c{rainy}, not prepare_rain_gear.
in_plum_rain_season.
rainy | cloudy :- in_plum_rain_season.
prepare_rain_gear :- &c{rainy}.

#show rainy/0.
#show cloudy/0.
#show prepare_rain_gear/0.