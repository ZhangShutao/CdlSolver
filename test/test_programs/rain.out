default model: 1
x:cloudy,prepare_rain_gear; y:cloudy,rainy,prepare_rain_gear;
default model: 2
x:rainy,prepare_rain_gear; y:rainy,prepare_rain_gear;